### ISSUE TITLE
add idaapi.get_inf_structure

### ISSUE THREAD

**Issue by williballenthin** [williballenthin/python-idb#25]
this will support the extraction of the processor use for a database.

definition of fields is here: https://www.hex-rays.com/products/ida/support/sdkdoc/structidainfo.html

**Comment by williballenthin** [comment]
looks like this is stored in netnode `Root Node` tag S index ??? (some number that looks like an address).

**Comment by williballenthin** [comment]
apparently this index is hardcoded. its the same in .idb files for /bin/ls and kernel32. also referenced here: https://github.com/nlitsme/idbutil/blob/ba686fb3449c05367ddb47a2ea8e683dafc6de23/idbtool.cpp#L359

and upon inspecting `ida.dll`, it is definitely is hardcoded:

```
.text:0000000010148490 sub_10148490 proc near
.text:0000000010148490
.text:0000000010148490 var_38= dword ptr -38h
.text:0000000010148490 var_28= word ptr -28h
.text:0000000010148490 var_26= byte ptr -26h
.text:0000000010148490 var_25= word ptr -25h
.text:0000000010148490 var_10= qword ptr -10h
.text:0000000010148490 var_8= qword ptr -8
.text:0000000010148490
.text:0000000010148490 ; __unwind { // __GSHandlerCheck
.text:0000000010148490 sub     rsp, 58h
.text:0000000010148494 mov     rax, cs:qword_105DD640
.text:000000001014849B xor     rax, rsp
.text:000000001014849E mov     [rsp+58h+var_10], rax
.text:00000000101484A3 mov     ecx, cs:root_node
.text:00000000101484A9 lea     r8, [rsp+58h+var_28]
.text:00000000101484AE mov     r9d, 16h
.text:00000000101484B4 mov     [rsp+58h+var_38], 'S'
.text:00000000101484BC mov     edx, 41B994h
.text:00000000101484C1 call    netnode_sup
```

**Comment by williballenthin** [comment]
the `idainfo` structure layout appears to have changed between 6.95 and 7.0. it'll take some work to figure out where things are supposed to be. let wait for someone to need to fetch these values before we sink too much time into this.

for right now, we just need to pull out `procname`.

**Comment by williballenthin** [comment]
`idaapi.get_inf_structure()` implemented in f8c610bef9fb7088d208e62011968ec075bb5d3f
